DeepLearningSuite/CMakeLists.txt

```diff
@@ -93,12 +93,12 @@ IF (NOT DARKNET_PATH)
 	MESSAGE(WARNING "DARKNET DISABLE")
 ENDIF()
 
-FIND_PATH( DARKNET_INCLUDE_DIR NAMES DarknetAPI/DarknetAPI.h  PATHS ${DARKNET_PATH}/include )
+FIND_PATH( DARKNET_INCLUDE_DIR NAMES DarknetAPI/DarknetAPI.h  PATHS ${DARKNET_PATH} )
 SET(DARKNET_NAMES darknetAPILib darknetExamplesLib darknetLib)
 #SET(DARKNET_NAMES darknetLib)
 
  FOREACH(N ${DARKNET_NAMES})
-	FIND_LIBRARY( DARKNET_LIBRARIES_${N} NAMES ${N} PATHS ${DARKNET_PATH}/lib )
+	FIND_LIBRARY( DARKNET_LIBRARIES_${N} NAMES ${N} PATHS ${DARKNET_INCLUDE_DIR}/DarknetAPI/ ${DARKNET_INCLUDE_DIR}/src/ ${DARKNET_INCLUDE_DIR}/examples)
```

Comment:
This directory structure is in the source repository of jderobot fork, you would only need this if your are linking libs from the source repository build.
And that's not how it is supposed to be, libraries are supposed to be linked from the installation directory not the source repository

Reply:
@vinay0410 Thanks a lot, I got your point.

Reply:
Great !!!